<?xml version="1.0"?>
<launch>
    <rosparam command ="load" file="$(find valid_grasp_generator)/parameter.yaml" />
    <node name="valid_grasp_generator" pkg="valid_grasp_generator" type="grasp_visualizer_and_saver.py" output="screen" />
</launch>
